apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -159,27 +169,27 @@ export const Editor = React.forwardRef<{ focus: () => void; blur: () => void },
       [fontFamily]
     );
 
-    const debouncedOnChange = useMemo(
-      () =>
-        debounce((value: string) => {
-          if (onChange) onChange(value);
-        }, 50),
```

Comment:
No need to debounce because it affects the input (editor) responsiveness and if typing quickly the changes are dropped and resulting in text "jumps". 
React is quick in doing updates, so we had to solve the root cause of the issue and not "hack" it.